mriqc/cli/run.py

```diff
@@ -189,7 +189,9 @@ def main():
         mod_group_reports = []
         for mod in config.execution.modalities or DEFAULT_TYPES:
             output_dir = config.execution.output_dir
-            dataframe, out_tsv = generate_tsv(output_dir, mod)
+            task_id = config.execution.task_id
+            task_id = task_id[0] if task_id else None
```

Comment:
I believe this may produce weird results if ``--task-id`` is set but `-m` is not, and the T1w images would get the task suffix.

```suggestion
            task_id = None
            if mod == "bold" and config.execution.task_id:
                  task_id = config.execution.task_id[0]
```

Why only the first task is picked?